product.php

```diff
@@ -0,0 +1,185 @@
+<?php include 'includes/header.php'; ?>
+
+
+<!--import css skeleton-->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css"/>
```

Comment:
The team working on the header and footer merged yesterday some changes to master. Could you pull those down, merge into your branch, and remove this line please :) The sitewide CSS was included as part of their changes and it includes Skeleton. Also do you have this branch up on Heroku somewhere so I can preview? Thanks!

You can merge this repos master into your forked master branch by using `git pull upstream master` and then push to origin. Lmk if you run into any issues or need help :)

Reply:
Code was removed